momento-sdk/src/intTest/java/momento/sdk/BaseCacheTestClass.java

```diff
@@ -15,22 +17,34 @@ public class BaseCacheTestClass {
   protected static CredentialProvider credentialProvider;
 
   protected static CacheClient cacheClient;
+  protected static CacheClient consistentReadCacheClient;
+  protected static CacheClient balancedReadCacheClient;
   protected static String cacheName;
 
   @BeforeAll
   static void beforeAll() {
+    final boolean consistentReads = System.getenv("CONSISTENT_READS") != null;
+
     credentialProvider = CredentialProvider.fromEnvVar("MOMENTO_API_KEY");
-    cacheClient =
-        CacheClient.builder(credentialProvider, Configurations.Laptop.latest(), DEFAULT_TTL_SECONDS)
-            .build();
+
+    final Configuration config = Configurations.Laptop.latest();
+    final Configuration consistentConfig = config.withReadConcern(ReadConcern.CONSISTENT);
+
+    balancedReadCacheClient =
+        CacheClient.builder(credentialProvider, config, DEFAULT_TTL_SECONDS).build();
+    consistentReadCacheClient =
+        CacheClient.builder(credentialProvider, consistentConfig, DEFAULT_TTL_SECONDS).build();
+    cacheClient = consistentReads ? consistentReadCacheClient : balancedReadCacheClient;
```

Comment:
Is there any reason to instantiate both clients here? Not a huge deal, but if you're only using one of the two it seems like you could just use the env var to determine which config to use and only ever instantiate a single cacheClient.

Reply:
That was done in other SDKs because we had tests that required the specific clients. I was on the fence about adding tests that explicitly check that the client works with each read concern, since they run with the read concern header on github and the canaries, but run without it locally. If I add those tests, we need the clients. Otherwise I'll remove them.